MetaMask/metamask-mobile
# refactor(ramp): refactor quotes view copy and list

## **Description**

This PR changes the copy and list of quotes in the Buy flow

## **Manual testing steps**
1. Go to buy flow
2. Select Payment Method, Input amount and Get Quotes
3. Confirm the quotes screen has the new copy and design

## **Screenshots/Recordings**

### **Before**
<img width="300" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/10b09545-610d-439f-9312-81570912d623" />


### **After**

<img width="300" src="https://github.com/MetaMask/metamask-mobile/assets/1024246/7f5ff4ae-197e-41b2-93b5-0789d5a84023" />


## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
